src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java

```diff
@@ -174,7 +178,11 @@ public void unsubscribe(@Nonnull EventFilter filter) {
                         subscribers.remove(filter);
                     }
                 }
-                publishStateEvent("unsubscribe");
+                publishStateEvent("unsubscribe",  new SimpleMessage()
+                        .set("sse_subs_dispatcher", id)
+                        .set("sse_subs_channel_name", channelName)
+                        .set("sse_subs_filter", filter.toJSON())
+                );
```

Comment:
It might help to define these three strings as static final Strings so they aren't duplicated and subject to typos in the future.